src/pages/_app.jsx

```diff
@@ -103,8 +103,8 @@ const WrappedApp = ({ Component, pageProps }) => {
 };
 
 WrappedApp.propTypes = {
-  Component: PropTypes.object.isRequired,
-  pageProps: PropTypes.string.isRequired,
+  Component: PropTypes.func.isRequired,
```

Comment:
is `PropTypes.element` more suitable?

Reply:
it has to be a func, otherwise we get a warning